.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules/
 tmp/
 npm-debug.log
 .idea
+.DS_Store
```

Comment:
this is developer specific (like the .idea above but that's not your fault) and should be in your .gitignore global file